src/clientdlg.cpp

```diff
@@ -948,7 +947,7 @@ void CClientDlg::SetMyWindowTitle ( const int iNumClients )
     {
         // if --clientname is used, the APP_NAME must be the very first word in
         // the title, otherwise some user scripts do not work anymore, see #789
-        strWinTitle += QString ( APP_NAME ) + " " + pClient->strClientName + " ";
+        strWinTitle += QString ( APP_NAME ) + " - " + pClient->strClientName + " ";
```

Comment:
This block (947 to 981) all seems a little clumsy to me and I have to re-read it several times to get the gist.  I think it probably wants a refactor at some point.  Not this change, though.